po/zh_CN.po

```diff
@@ -19904,8 +19903,8 @@ msgid ""
 "changes before moving to a new transaction, discard the changes, or return "
 "to the changed transaction?"
 msgstr ""
-"当前的交易已被修改。您想在移到新的交易前保存修改、忽略修改，还是返回已改变的"
-"交易？"
+"当前的交易事项已被修改。您想在移到新的交易事项前保存修改、忽略修改，还是返回已改变的"
+"交易事项？"
 
 #. Translators: The 'sample:' items are
 #. strings which are not displayed, but only
```

Comment:
Did you read this comment?

For me as german speaking it is always annoying if the samples are nor set, because a german text is usually longer than the respective english text and then input fields become too small to read the entered text or you have to waste time with resizing of columns. So please use a typical chinese text behind "sample:" below.

Reply:
Missed that. Lucky, chinese is usually shorter. Fixed all.